src/quacc/wflow_tools/decorators.py

```diff
@@ -616,6 +619,9 @@ def wrapper(
     ):
         return func(*f_args, **f_kwargs)
 
+    if hasattr(func, "__changed__") and func.__changed__:
```

Comment:
We can probably use `getattr` with a default here again, if I'm understanding `getattr` corectly.